ansible_collections/arista/avd/plugins/filter/esi_management.py

```diff
@@ -1,11 +1,8 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-from jinja2 import TemplateError
-from ansible.errors import AnsibleFilterError
 import re
 
-
```

Comment:
As per PEP8 rules, we should have 2 blank lines between import and rest of code:

- Two blank lines between the import statements and other code.
- Two blank lines between each function.

https://legacy.python.org/dev/peps/pep-0008/#blank-lines